.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,9 @@
+- id: helm_docs
+  args: []
+  description: Uses 'helm-docs' to create documentation from the Helm chart's 'values.yaml' file, and inserts the result into a corresponding 'README.md' file.
+  entry: scripts/helm_docs.sh
+  exclude: charts\/.*$
+  files: (Chart\.yaml)$
```

Comment:
You will probably want: `(Chart|requirements)\.yaml$`

Reply:
Done